flintrock/services.py

```diff
@@ -106,11 +115,18 @@ def health_check(
         """
         raise NotImplementedError
 
+    def create_security_groups(self, flintrock_client_cidr: str):
+        """
+        Create and return the EC2 SecurityGroupRules required by this service
+        """
+        return []
```

Comment:
Why `return []` and not `raise NotImplementedError`? I would prefer the latter for consistency, unless there is some reason to do otherwise.

Reply:
I was torn between the two at the time to be honest. I've changed it as requested.